distribution/kernel/pom.xml

```diff
@@ -165,6 +170,12 @@
                                     <destFileName>ddf-branding.jar</destFileName>
                                 </artifactItem>
                                 <artifactItem>
+                                    <groupId>org.ops4j.pax.logging</groupId>
+                                    <artifactId>pax-logging-log4j2</artifactId>
+                                    <version>${pax.logging.version}</version>
+                                    <outputDirectory>${setup.folder}/system/org/ops4j/pax/logging/pax-logging-log4j2/${pax.logging.version}</outputDirectory>
```

Comment:
I did some research and I believe we can eliminate this and build up the startup.properties file using the karaf-maven-plugin.

http://karaf.apache.org/manual/latest/#_maven_assembly